### DISCUSSION TITLE
What is that?

### DISCUSSION THREAD

**Discussion by niveKinomaL** [mdbootstrap/TW-Elements#2009] [Support from community]
![image](https://github.com/mdbootstrap/TW-Elements/assets/83254979/78823a8b-9720-4751-bffc-1225146c5ee6)
How to remove that blue ring or border idk in te elements, i tried border-none outline-none is from date picker

**Comment by iprzybysz** [comment]
Hi, these are notches from the input component. Check out our [Input Classes documentation](https://tw-elements.com/docs/standard/forms/inputs/#api-section-classes). You can remove this border using class customization. 

Here is the example code:

```
<div
  class="relative mb-3"
  data-te-datepicker-init
  data-te-input-wrapper-init
  data-te-class-notch-leading-normal="border-neutral-300 dark:border-neutral-600"
  data-te-class-notch-middle-normal="border-neutral-300 dark:border-neutral-600"
  data-te-class-notch-trailing-normal="border-neutral-300 dark:border-neutral-600"
>
  <input
    type="text"
    class="peer block min-h-[auto] w-full rounded border-0 bg-transparent px-3 py-[0.32rem] leading-[1.6] outline-none transition-all duration-200 ease-linear focus:placeholder:opacity-100 peer-focus:text-primary data-[te-input-state-active]:placeholder:opacity-100 motion-reduce:transition-none dark:text-neutral-200 dark:placeholder:text-neutral-200 dark:peer-focus:text-primary [&:not([data-te-input-placeholder-active])]:placeholder:opacity-0"
    placeholder="Select a date"
   />
  <label
    for="floatingInput"
    class="pointer-events-none absolute left-3 top-0 mb-0 max-w-[90%] origin-[0_0] truncate pt-[0.37rem] leading-[1.6] text-neutral-500 transition-all duration-200 ease-out peer-focus:-translate-y-[0.9rem] peer-focus:scale-[0.8] peer-data-[te-input-state-active]:-translate-y-[0.9rem] peer-data-[te-input-state-active]:scale-[0.8] motion-reduce:transition-none dark:text-neutral-200"
    >Select a date</label
  >
</div>
```

**Comment by hallelujah** [comment]
It is default browser outline, nothing to do with tw-elements I think. In tailwindcss you can add `focus-visible:outline-none`
Using `outline-none` directly is not really recommended

[Removing outlines](https://tailwindcss.com/docs/outline-style#removing-outlines)

>Use outline-none to hide the default browser outline on focused elements.
> It is highly recommended to apply your own focus styling for accessibility when using this utility.

**Reply by iprzybysz** [reply]
I showed you above how to delete our notches by class customization - please checkout my previous message.
What the end result you want to achieve? - I cannot recreate the issue you are experiencing. Is there any chance to show us the code you are using? How do you want your input to look like? The border on your screenshot looks like our notch.

**Reply by hallelujah** [reply]
@iprzybysz ? I am trying to help, I am not the original poster...

**Reply by iprzybysz** [reply]
I apologize for any confusion. I had mistakenly assumed that I was conversing with the original author of the problem. It was an oversight on my part, and I am truly sorry for any misunderstanding.
